.gitlab-ci.yml

```diff
@@ -59,14 +55,10 @@ stages:
       Build dependencies and generate host-config file (uberenv)"
     - ${EXEC_PREFIX} ${RESOURCES} scripts/llnl/build_tpls.py --spec=${SPEC} --directory=${CI_PROJECT_NAME}
     - echo -e "section_end:$(date +%s):dependencies\r\e[0K"
-    #BUILD
-    - RESOURCES="-t 5 -N 1"
-    - echo -e "Build ${CI_PROJECT_NAME}"
-    - ${EXEC_PREFIX} ${RESOURCES} scripts/gitlab/build_and_test.sh --build-only
-    #TEST
-    - RESOURCES="-t 5 -N 1"
-    - echo -e "Test ${CI_PROJECT_NAME}"
-    - ${EXEC_PREFIX} ${RESOURCES} scripts/gitlab/build_and_test.sh --test-only
+    #BUILD + TEST
+    - RESOURCES="-t 10 -N 1"
+    - echo -e "Build and test ${CI_PROJECT_NAME}"
+    - ${EXEC_PREFIX} ${RESOURCES} scripts/llnl/build_src.py
```

Comment:
host-config here for consistency...

Reply:
the quartz build_with_deps jobs can figure out their host-configs from the hostname/sys_type and don't have host-configs explicitly specified in the CI - should I add them?

Reply:
Is it finding it through the environment variable? Because there are two different host-configs on this system we test.